pyzx/rules.py

```diff
@@ -251,6 +251,75 @@ def unspider(g: BaseGraph[VT,ET], m: List[Any], qubit:FloatInt=-1, row:FloatInt=
         g.set_phase(u, 0)
     return v
 
+MatchWType = Tuple[VT,VT]
+
+def match_w_fusion(g: BaseGraph[VT,ET]) -> List[MatchWType[VT]]:
+    """Does the same as :func:`match_spider_parallel` but with ``num=1``."""
+    return match_spider_parallel(g, num=1)
```

Comment:
Is this correct?

Reply:
Shouldn't this be match_w_spider_parallel or something like that?

Reply:
Good catch. I copied it from match_spider but forgot to modify it.